src/nvim/os/input.c

```diff
@@ -367,12 +371,14 @@ static uint8_t check_multiclick(int code, int grid, int row, int col)
   orig_mouse_row = row;
 
   uint8_t modifiers = 0;
-  if (orig_num_clicks == 2) {
-    modifiers |= MOD_MASK_2CLICK;
-  } else if (orig_num_clicks == 3) {
-    modifiers |= MOD_MASK_3CLICK;
-  } else if (orig_num_clicks == 4) {
-    modifiers |= MOD_MASK_4CLICK;
+  if (code != KE_MOUSEMOVE) {
```

Comment:
This condition is not needed. When `code == KE_MOUSEMOVE` the function has already returned above.

Reply:
It only returns above when there has been no move. When there has been a move it continues on since the current position must be saved.

After saving the current position there could be
```
if (code == KE_MOUSEMOVE) { return 0; }
```
but that looked messier.

Reply:
Oops